charts/calhoun/templates/role.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Values.name }}-service-role
+  labels:
+    {{- include "calhoun.labels" . | nindent 4 }}
+rules:
+- apiGroups: ['policy']
+  resources: ['podsecuritypolicies']
+  verbs:     ['use']
+  resourceNames:
+  - terra-default-psp
+- apiGroups: ['']
+  resources: ['secrets']
+  verbs: ['get', 'create']
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["extensions", "apps"]
+  resources: ["deployments"]
+  verbs: ["get", "list", "watch"]
```

Comment:
If calhoun doesn't need to make calls to the api of the k8s cluster it is running in. we can get rid of these. They are specific to some requirement of buffer service.
```suggestion
```